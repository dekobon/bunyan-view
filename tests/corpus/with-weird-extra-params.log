{"name":"agent1","pid":73267,"req":null,"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"req":3.14,"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"req":"some-string","hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"req":["a","b"],"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"req":{"a":"b"},"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"req":{"a":{"b":"c"}},"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"req":{"a":["b","c"]},"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"wat":null,"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"wat":3.14,"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"wat":"some-string","hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"wat":["a","b"],"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"wat":{"a":"b"},"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"wat":{"a":{"b":"c"}},"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"wat":{"a":["b","c"]},"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"err":true,"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"err":false,"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"err":null,"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"err":3.14,"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"err":"some-string","hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"err":["a","b"],"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"err":{"a":"b"},"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"err":{"a":{"b":"c"}},"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}
{"name":"agent1","pid":73267,"err":{"a":["b","c"]},"hostname":"headnode","level":30,"msg":"message","time":"2012-05-08T16:57:55.586Z","v":0}