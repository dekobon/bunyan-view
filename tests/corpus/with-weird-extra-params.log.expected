[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message (req=null)
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message (req=3.14)
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message (req=some-string)
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message
    req: [
      "a",
      "b"
    ]
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message (req.a=b)
    undefined undefined HTTP/1.1
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message
    undefined undefined HTTP/1.1
    --
    req.a: {
      "b": "c"
    }
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message
    undefined undefined HTTP/1.1
    --
    req.a: [
      "b",
      "c"
    ]
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message (wat=null)
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message (wat=3.14)
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message (wat=some-string)
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message
    wat: [
      "a",
      "b"
    ]
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message
    wat: {
      "a": "b"
    }
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message
    wat: {
      "a": {
        "b": "c"
      }
    }
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message
    wat: {
      "a": [
        "b",
        "c"
      ]
    }
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message (err=null)
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message (err=3.14)
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message (err=some-string)
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message
    err: [
      "a",
      "b"
    ]
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message (err.a=b)
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message
    err.a: {
      "b": "c"
    }
[2012-05-08T16:57:55.586Z]  INFO: agent1/73267 on headnode: message
    err.a: [
      "b",
      "c"
    ]
